docs/migration-guide.md

```
@@ -1,36 +1,34 @@
 ---
 title: "Migration Guide — 遷移指南"
 tags: [migration, getting-started]
-audience: [tenant, devops]
+audience: [tenant, devops, platform-engineer, sre]
 version: v2.9.0
 lang: zh
 ---
 # Migration Guide — 遷移指南
 
 > **Language / 語言：** **中文 (Current)** | [English](./migration-guide.en.md)
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)